tests/console/perl_bootloader.pm

```diff
@@ -20,17 +19,38 @@ use version_utils 'is_sle';
 sub run {
     my ($self) = @_;
     select_serial_terminal;
+    zypper_call 'in perl-Bootloader';
 
-    if (script_run 'rpm -q perl-Bootloader' == 1) {
-        zypper_call 'in perl-Bootloader';
+    # Set bootloader to grub2 and check if it was changed in /etc/sysconfig/bootloader
+    if (is_sle(">=15-SP3")) {
```

Comment:
Does this actually do anything? GRUB2 is default already.

This will most likely also fail if EFI is used.

Reply:
Does this only apply to SLE? is_sle is false on sle_micro, ALP, Tumbleweed, Leap, ...

Reply:
We tested it on SLE, TW and Leap, but Alp and Micro require additional work.
Cedric is new, IMO it can be done later. There is also issue with `install_package` https://progress.opensuse.org/issues/153865

Reply:
The tests on Alp and Micro will be done later: https://progress.opensuse.org/issues/153871